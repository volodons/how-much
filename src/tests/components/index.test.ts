import {
    CurrencyConverter,
    CurrentCurrency,
    ExchangeRates,
    Navigation,
    Title,
} from '../../components/index';

describe('index.ts', () => {
    it('should export CurrencyConverter component', () => {
        expect(CurrencyConverter).toBeDefined();
    });

    it('should export CurrentCurrency component', () => {
        expect(CurrentCurrency).toBeDefined();
    });

    it('should export ExchangeRates component', () => {
        expect(ExchangeRates).toBeDefined();
    });

    it('should export Navigation component', () => {
        expect(Navigation).toBeDefined();
    });

    it('should export Title component', () => {
        expect(Title).toBeDefined();
    });
});
