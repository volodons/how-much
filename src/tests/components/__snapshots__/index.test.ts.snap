// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index.ts should export CurrencyConverter component 1`] = `[Function]`;

exports[`index.ts should export CurrentCurrency component 1`] = `[Function]`;

exports[`index.ts should export ExchangeRates component 1`] = `[Function]`;

exports[`index.ts should export Navigation component 1`] = `[Function]`;

exports[`index.ts should export Title component 1`] = `[Function]`;
